src/pybind/mgr/dashboard/tests/test_docs.py

```diff
@@ -30,10 +30,14 @@ class DecoratedController(RESTController):
         },
     )
     @Endpoint(json_response=False)
-    @RESTController.Resource('PUT')
+    @RESTController.Resource('PUT', version='0.1')
```

Comment:
In the parent backport this has been removed:
https://github.com/ceph/ceph/blob/22c79fa7ffe06c29da5b0510dfd3d71293c397ca/src/pybind/mgr/dashboard/tests/test_docs.py#L4

Is it kept here intentionally?

Reply:
Just removed that now.